codalab/worker_manager/worker_manager.py

```diff
@@ -151,17 +152,20 @@ def run_loop(self):
             try:
                 self.run_one_iteration()
             except (
-                urllib.error.URLError,
                 http.client.HTTPException,
                 socket.error,
-                NotFoundError,
+                urllib.error.URLError,
                 JsonApiException,
+                ssl.SSLError,
             ):
                 # Sometimes, network errors occur when running the WorkerManager . These are often
                 # transient exceptions, and retrying the command would lead to success---as a result,
                 # we ignore these network-based exceptions (rather than fatally exiting from the
                 # WorkerManager )
                 traceback.print_exc()
+            except NotFoundError:
```

Comment:
Bit confused why you don't check for LoginPermissionError -- what's the difference between that error and NotFoundError in this context?